components/policy-mgt/org.wso2.carbon.policy.mgt.core/src/main/java/org/wso2/carbon/policy/mgt/core/dao/impl/ProfileDAOImpl.java

```diff
@@ -155,6 +155,29 @@ public boolean deleteProfile(Profile profile) throws ProfileManagerDAOException
     }
 
     @Override
+    public boolean deleteProfile(int profileId) throws ProfileManagerDAOException {
```

Comment:
This boolean is used just to acknowledge whether the operation is successful. It returns true if there are any affected rows of the query.